code/WorkInProgress/Electronics.dm

```diff
@@ -466,92 +462,253 @@
 
 	var/processing = 0
 	var/net_id = null
-	var/frequency = 1149
+	var/frequency = 1467
```

Comment:
Can this be made a constant? Though I think everything is currently documented in comments...

Reply:
A constant? Where?
Currently no one has sat down and written defines for frequencies which is really unfortunate

Reply:
https://github.com/goonstation/goonstation/blob/7f51873c5dbe6b80d3567f1b8121bc16729a18ce/_std/defines/_frequency.dm#L4

https://github.com/goonstation/goonstation/blob/7f51873c5dbe6b80d3567f1b8121bc16729a18ce/code/datums/communications.dm#L4